### ISSUE TITLE
Vox Raiders is the worst gamemode

### ISSUE THREAD

**Issue by TheFriggenSawbones** [d3athrow/vgstation13#12112]
It's really fuckin bad. 

It's, in theory, super stealth nuke ops. Cool idea but the way its currently implemented is very bad. It expects both the raiders and the crew to not instantly valid eachother.

In the last round I played the raiders got found it and the AI located where the shuttle was and sent borgs after the vox. Vox mostly died and the last one went back into deepspace ending the round. We could fix some of these issues by making like admin clarifications of certain things (I.E Is the raider shuttle the same as the Nuke Ops where you can't actively hunt for it? Are raiders totally valid? How do you know they're not a trader?) Overall no one has fun and it's a bad choice for a primary gamemode.

**Comment by Exxion** [comment]
On rare occasions it's very fun for the raiders, but it's usually mediocre-bad for most of the players and it's _never_ fun for everyone
It's one of those gamemodes that probably works a lot better on a higher-RP server

**Comment by CptWad** [comment]
I mean new shit could help
how about instead of c4 they get some item that drills into walls to create a wall that players can walk through but air cant in some neat scifi way
then maybe make cloaking devices work
super secret birb sneak force

**Comment by Allspice** [comment]
About 75% of the time the jectives are something dumb like guns and mats or the PA and mats. If the birds can get their shit together for 3 seconds they don't interact with anybody but themselves for 20 minutes then the round ends, how antagonizing.

**Comment by jakmak** [comment]
Raiders have some serious issues on so many levels. 
1. Borg's. A single sillicons could easily kill all the raiders if they get on the ship and the vox have NO ways to defend from it. Yes they have ion guns and a few ion grenades BUT ions do NOTHING to Borg's meaning they are helpless to them.
2. Communication barrier. Raiders cannot speak galactic common removing ANY attempt at peacefull negotiation. Yes they could get a trader to be a translator but that usually turns into a total mess.
3. Valids. During raiders the best I've seen a group of people work together in any game was the station during raiders. They all know there isn't anyone who would betray them so they give everyone all access and guns to kill the 6 mostly harmless space Mexicans. Have auto traitor run in the background to prevent the crew from going LETS GET THEM VALIDS! 
4. Raiders themselves. There are 6 raiders who have a objective of leave no man behind. There is a good chance 4 of those 6  birds have ADHD, autism, and down syndrome. What happens is a retard goes out and gets caught or he gets left behind and tells the team he is still at the trade post when the team is welding into armory. Reduce the amount of raiders to 4 or maybe even 3.
5. Their equipment. The birds get 3 tazer, 3 USELESS ion guns, some shitty glorified chloral syringe guns which if used on someone in a hardsuit probably would result in a dart not penetrating it and most of your encounters will be in space meaning the enemy always has hardsuits, a cloner with enough biomass for a single bird and no way to heal the post cloning retardation meaning they are effectively out of the round, A useless RCD, a prison with no 02 tanks and a windoor they could just smash open with the air tank they need to survive,  a single emag that always goes missing and a ship that can go to 3 locationss around the station and 1 place off of Z1. Make the door in bird jail a proper airlock with agent Id access only along with a o2 tank and mask, give the raiders ship more places to fly around and also allow it to fly to the middle of nowhere so they can hide, buff their dart guns and give them some rezadone pills so any cloned vox are not useless to the team.
6.  The Gamemode itself. Once raiders fly home round is over. Instead remove that so raiders once objectives are done they could drop them off at the base and could return to station do more fun stuff other than mining for 20 gold and to prevent it from turning to shitter safari have auto traitor running in the backround so it doesn't get boring on station.

**Comment by ghost** [comment]
Throw it in with autotraitor or something

Group it with another gamemode at least

**Comment by Ephx1** [comment]
> ions do NOTHING to Borgs

What fucking drugs are you on?
No, seriously, tell me.

Ion grenades are miserable sure, but you also have emags, stun, emag spam, crowbar, emag spam.

**Comment by jakmak** [comment]
@ephx1for normal tators that might work but you have to remember 90% of the time the emag goes missing. And with ion grenades absolute best case scenario you get 10 seconds of stun but you are more likely to get 1-5 seconds of stun or 5-10 if your lucky. And if you screw up the emag process or fumble a tool for half a second that Borg will come out of it and beat your ass. God the effect ions have on sillicons needs to be buffed massively.

**Comment by Shadowmech88** [comment]
Don't raiders get ion rifles? Which are capable of completely draining a borg's cell if you unload into it?

**Comment by Allspice** [comment]
The borg is rendered nonfunctional through other means long before the cell drain is noticed.

**Comment by Ephx1** [comment]
> remember 90% of the time the emag goes missing

Sounds like you or other teammates are just shit.

> God the effect ions have on sillicons needs to be buffed massively

God, I really want to hit you for being so stupid. Ion grenades are bad, but you don't even NEED TO FUCKING AIM with an Ion Rifle.

**Comment by Durgen** [comment]
Raiders should be a random event that pulls from ghosts from deadchat, and not a gamemode

It's snoozeville for all involved

**Comment by kernikov** [comment]
TL;DR, HEIST IS STILL BAD.
CRYO PRISON TUBES WITH 6 KIDNAP OBJECTIVES WHEN

**Comment by Probe1** [comment]
That is pretty wretched gameplay to be stuck in a tube.

**Comment by SonixApache** [comment]
the obvious answer is to remove vox


bird ops should be an event, not a gamemode;
- poll ghosts
- send birds
- give birds respawn ticket if they succeed
- put bird ops on roundend report

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
This title did not age well
